crates/router/src/services/authentication.rs

```diff
@@ -88,6 +88,11 @@ pub enum AuthenticationType {
         merchant_id: id_type::MerchantId,
         user_id: Option<String>,
     },
+    MerchantJwtWithProfileId {
+        merchant_id: id_type::MerchantId,
+        profile_id: Option<id_type::ProfileId>,
+        user_id: Option<String>,
```

Comment:
I don't think `Option` is needed here.

Reply:
This ProfileId should come from JWT token right? 
ProfileId is optional in JWT.

Reply:
I was referring to user_id. Sorry for the confusion.